lib/internal/process/esm_loader.js

```diff
@@ -60,17 +60,24 @@ async function initializeLoader() {
     cwd = 'file:///';
   }
 
+  const keyedExportsList = [];
+
   // A separate loader instance is necessary to avoid cross-contamination
   // between internal Node.js and userland. For example, a module with internal
   // state (such as a counter) should be independent.
   const internalEsmLoader = new ESMLoader();
 
-  // Importation must be handled by internal loader to avoid poluting userland
-  const keyedExportsList = await internalEsmLoader.import(
-    customLoaders,
-    pathToFileURL(cwd).href,
-    ObjectCreate(null),
-  );
+  for (const customLoader of customLoaders) {
```

Comment:
To avoid unsafe array iteration
```suggestion
  for (let i = 0; i < customLoaders.length; i++) {
    const customLoader = customLoaders[i];
```